src/platform/datapath_raw_socket.c

```diff
@@ -356,8 +356,31 @@ CxPlatResolveRoute(
     CXPLAT_DBG_ASSERT(IpnetRow.PhysicalAddressLength == sizeof(Route->NextHopLinkLayerAddress));
     CxPlatCopyMemory(&Route->NextHopLinkLayerAddress, IpnetRow.PhysicalAddress, sizeof(Route->NextHopLinkLayerAddress));
 
-    Route->QueueId = 0; // TODO - Any way figure this out upfront?
 
+    //
+    // Find the interface that matches the route we just looked up.
+    //
+    CXPLAT_LIST_ENTRY* Entry = Socket->Datapath->Interfaces.Flink;
```

Comment:
nit: put this into the `for` loop since you don't need it outside the scope

Reply:
Oh, I placed it outside so I don't need to break `for` loop into multiple lines, which seems ugly 😆